types/types.tsx

```diff
@@ -117,6 +128,20 @@ export type TDropdownProps = {
 	onSelect:
 		| React.Dispatch<React.SetStateAction<TDropdownOption>>
 		| ((option: TDropdownOption) => void);
+	balances?: {
+		[key: string]: TBalanceData
+	};
+};
+
+export type TDropdownItemProps = {
+	option: TDropdownOption;
+	onSelect:
+		| React.Dispatch<React.SetStateAction<TDropdownOption>>
+		| ((option: TDropdownOption) => void);
```

Comment:
```suggestion
	onSelect: (option: TDropdownOption) => void;
```
I think we should a bit more generic here